src/HashBus.Projector.TopRetweeters/TopRetweetersProjection.cs

```diff
@@ -19,17 +21,17 @@ public TopRetweetersProjection(IRepository<string, IEnumerable<Retweet>> tweets)
             this.tweets = tweets;
         }
 
-        public void Handle(Application.Events.TweetAnalyzed message)
+        public Task Handle(TweetAnalyzed message, IMessageHandlerContext context)
         {
             if (message.Tweet.RetweetedTweet == null)
             {
-                return;
+                return Task.FromResult(0);
             }
 
             var trackTweets = this.tweets.GetAsync(message.Tweet.Track).GetAwaiter().GetResult().ToList();

```

Comment:
make the Handle method `async` and remove `.GetAwaiter().GetResult()`, making it:

```
( await this.tweets.GetAsync(message.Tweet.Track) ).ToList();
```

then you can remove all the `return Task.FromResult(0);`